examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
+  <Catalog
+    configuration={{
```

Comment:
We also need to keep in mind that the standalone usage should still work as easily as it does now, without additionally importing React or anything.